src/tex/ms.tex

```diff
@@ -133,7 +134,6 @@ \subsection{Model evaluation}
     \centering
     \includegraphics{figures/human_subset_performance.pdf}
     \caption{\textbf{Performance of models and humans on the \enquote{tiny} subset of the  \chembench corpus.} The figure shows the percentage of questions that the models answered correctly. We use horizontal bars to indicate the performance of various models and highlight statistics of the human performance.
```

Comment:
```suggestion
    \caption{\textbf{Performance of models and humans on the \enquote{human} subset of the  \chembench corpus.} The figure shows the percentage of questions that the models answered correctly. We use horizontal bars to indicate the performance of various models and highlight statistics of the human performance.
```

Reply:
not sure what nomenclature we should use - we call it "tiny" in other places (with reference to the "tiny benchmarks" paper)

Reply:
> I'd go for human everywhere, since the purpose of that subset was to evaluate the humans

yes and no. The purpose also was to to really have this smaller subset you can run during the training. This is not feasible with the complete ChemBench corpus.